docs/user/security/authorization/index.asciidoc

```diff
@@ -6,7 +6,21 @@ The Elastic Stack comes with the `kibana_admin` {ref}/built-in-roles.html[built-
 
 When you assign a user multiple roles, the user receives a union of the roles’ privileges. Therefore, assigning the `kibana_admin` role in addition to a custom role that grants {kib} privileges is ineffective because `kibana_admin` has access to all the features in all spaces.
 
-NOTE: When running multiple tenants of {kib} by changing the `kibana.index` in your `kibana.yml`, you cannot use `kibana_admin` to grant access. You must create custom roles that authorize the user for that specific tenant. Although multi-tenant installations are supported, the recommended approach to securing access to {kib} segments is to grant users access to specific spaces.
+[[xpack-security-multiple-tenants]]
+==== Supporting multiple tenants
+
+There are two approaches to supporting multi-tenancy in {kib}:
+
+1. *Recommended:* create a space and a limited role for each tenant, and configure each user with the appropriate role.
```

Comment:
```suggestion
1. *Recommended:* Create a space and a limited role for each tenant, and configure each user with the appropriate role.
```

Reply:
Are there instructions somewhere that we can point to?

Reply:
> Are there instructions somewhere that we can point to?

Yes, we can point to the "Securing access to Kibana" tutorial; it covers creating Spaces, Roles, _and_ Users :+1: